### PR TITLE
Update CONTRIBUTING.md documentation with common issues section

### PR DESCRIPTION
**Pull request by kamperiadis**
Updated CONTRIBUTING.md documentation with "Common Issues" section.

### Issue describing the changes in this PR

As part of onboarding, I have found that some information is missing and having it documented would help contributors in the future. 

### Pull request checklist

* [x] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [x] My changes **should not** be added to the release notes for the next release
  * [ ] Otherwise: I've added my notes to `release_notes.md`
* [x] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -171,6 +171,14 @@ With this you have completed the setup for Azure Functions Host and the Azure Fu
 
   - Restart host debugger
 
+### Common Issues
+#### Extension Bundle
+If you get any exceptions from the Host that are similar to this "The binding type(s) were not found in the configured extension bundle. Please ensure the type is correct and the correct version of extension bundle is configured", then you need to follow the steps here:
+https://github.com/Azure/azure-functions-host/wiki#extension-bundle-configuration
+
+#### App settings
+If you get any exceptions from the Host indicating that an app setting is not defined even though it has been defined in Azure for the function app, as well as in the local.settings.json file, this is because you need to add the app setting as one of the Host's environment variables (in launch.json)
```
Comment (kaibocai) [inline CONTRIBUTING.md]: this really depends on how you are debugging your function app on your local, if use are using the host code to debug then you should add appsettings in launch.json file in the env part. If on there other hand, if you are debugging from core tools then you should add your appsettings to the local.setting.json file in your function app.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -171,6 +171,14 @@ With this you have completed the setup for Azure Functions Host and the Azure Fu
 
   - Restart host debugger
 
+### Common Issues
+#### Extension Bundle
+If you get any exceptions from the Host that are similar to this "The binding type(s) were not found in the configured extension bundle. Please ensure the type is correct and the correct version of extension bundle is configured", then you need to follow the steps here:
```
Comment (kaibocai) [inline CONTRIBUTING.md]: probably add an tip for what value should be given to download path as it's confusing. 

Please add below context as well. 
For normal http trigger we don't need extension bundle, but for other type of triggers like eventhub, etc, then we need to enable extension bundle. I remember you don't need this step by debugging through core tools can you please confirm. Thanks.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -171,6 +171,14 @@ With this you have completed the setup for Azure Functions Host and the Azure Fu
...
+https://github.com/Azure/azure-functions-host/wiki#extension-bundle-configuration
```
Comment (shreyas-gopalakrishna) [inline CONTRIBUTING.md:186]: Recommend adding a sample `host.json` config here that can be reused.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -171,6 +171,14 @@ With this you have completed the setup for Azure Functions Host and the Azure Fu
...
+
+#### App settings
```
Comment (shreyas-gopalakrishna) [inline CONTRIBUTING.md]: I suggest we move this section from common issues to `Testing a new version of the Java Worker` section. We can document the different ways to debug including the instructions for using app settings.

**Review by shreyas-gopalakrishna — APPROVED** [review]

**Review by kaibocai — APPROVED** [review]
LGTM! Thanks!

**Comment by kaibocai** [comment]
@kamperiadis , try create the PR under the target repo, this issue we saw before that it happens when you create PR from a forked repo. Thanks.

**Comment by kamperiadis** [comment]
New PR: https://github.com/Azure/azure-functions-java-worker/pull/695

**Pull request closed without merging**